code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -624,26 +624,28 @@
 	else if (iscrowbar(W))	// crowbar means open or close the cover
 		if(opened)
 			if(cell)
-				user << "You close the cover."
-				opened = 0
-				updateicon()
+				user.visible_message("<span class='notice'>\The [user] begins clasping shut \the [src]'s maintenance hatch.</span>", "<span class='notice'>You begin closing up \the [src].</span>")

```

Comment:
Something about the phrasing here seems off, though it's fine as-is.

Reply:
perhaps "to close" instead of "clasping shut"? pretty sure it's held closed by screws.

Reply:
It is not held closed by screws because you use a crowbar to open it.